### PR TITLE
restore-scm api dep

### PR DESCRIPTION
**Pull request by michaelneale**
this was passing when on keiths fork. Branches are easier to see what was the difference before merging so this is just a guess.

### CODE REVIEW

File: blueocean-rest-impl/pom.xml [hunk 1]
```diff
@@ -35,6 +35,10 @@
             <groupId>org.jenkins-ci.plugins</groupId>
             <artifactId>display-url-api</artifactId>
         </dependency>
+         <dependency>
+            <groupId>org.jenkins-ci.plugins</groupId>
+            <artifactId>scm-api</artifactId>
```
Comment (vivek) [inline blueocean-rest-impl/pom.xml]: 🐛  We can't add it here. scm api shouldn't be in this module. Some other class path issue (sigh)... will get to it. Lets do the right way.

Comment (michaelneale) [reply blueocean-rest-impl/pom.xml]: @vivek this is just to see if it changes the result of the failing test on master... which seems a brittle test

Comment (vivek) [reply blueocean-rest-impl/pom.xml]: ah ok 👍

Comment (michaelneale) [reply blueocean-rest-impl/pom.xml]: so - as you can see @vivek this made it pass by probably changing how the test class is loaded :(

**Pull request closed without merging**